hugegraph-dist/pom.xml

```diff
@@ -168,51 +144,162 @@
                     </filesets>
                 </configuration>
             </plugin>
-
-            <plugin>
-                <artifactId>maven-antrun-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>download-swagger-ui</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
-                        <configuration>
-                            <tasks>
-                                <echo file="${project.basedir}/dist.sh">
-                                    wget --version 1>/dev/null || exit
-                                    wget https://github.com/swagger-api/swagger-ui/archive/refs/tags/v4.15.5.tar.gz
-                                    tar zxvf v4.15.5.tar.gz
-                                    echo "window.onload = function() { window.ui = SwaggerUIBundle({
-                                    url:'/openapi.json',dom_id:'#swagger-ui',deepLinking:true,layout:'StandaloneLayout',
-                                    presets:[SwaggerUIBundle.presets.apis, SwaggerUIStandalonePreset ],
-                                    plugins:[SwaggerUIBundle.plugins.DownloadUrl]});};" > swagger-ui-4.15.5/dist/swagger-initializer.js
-                                    cp -r swagger-ui-4.15.5/dist ../${final.name}/swagger-ui
-                                </echo>
-                                <exec executable="${shell-executable}" dir="${project.basedir}" failonerror="true">
-                                    <arg line="./dist.sh"/>
-                                </exec>
-                            </tasks>
-                        </configuration>
-                    </execution>
-                    <execution>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
-                        <configuration>
-                            <target>
-                                <tar destfile="${final.destfile}" compression="gzip">
-                                    <tarfileset dir="${top.level.dir}/" filemode="755">
-                                        <include name="${final.name}/**"/>
-                                    </tarfileset>
-                                </tar>
-                            </target>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
         </plugins>
+
+        <pluginManagement>
```

Comment:
need `pluginManagement`?

Reply:
> need `pluginManagement`?
> 
> BTW, all the CI failed now 🔎

1. for reuse the configuration of the `maven-antrun-plugin`
2. because of https://github.com/apache/incubator-hugegraph/pull/2277#discussion_r1302502303, fix in [`880d283` (#2277)](https://github.com/apache/incubator-hugegraph/pull/2277/commits/880d28358c7c79e71caba2424b70696606ecc9e7)